handler.go

```diff
@@ -126,7 +136,30 @@ func (h *Handler) methodNotAllowedHandler(w http.ResponseWriter, r *http.Request
 
 // ServeHTTP handles an HTTP request.
 func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	t := time.Now()
 	h.Router.ServeHTTP(w, r)
+	dif := time.Since(t)
+
+	// Handle some stats tagging
+	statsTags := make([]string, 0, 3)
+
+	if h.Cluster.LongQueryTime > 0 && dif > h.Cluster.LongQueryTime {
```

Comment:
let's revisit how the changes in `ServerHTTP` are implemented. these changes panic if `h.Cluster` or `h.Holder` are `nil`, which might be the case (at least in tests). See test `TestHandler_Query_Params_Err` as an example. Perhaps there's a cleaner way to indicate whether stats in enabled.